### ISSUE TITLE
regression: play-ws failing in 2.12.x build

### ISSUE THREAD

**Issue by SethTisue** [scala/community-builds#564]
seen at e.g. https://scala-ci.typesafe.com/view/All/job/scala-2.12.x-integrate-community-build/1612/consoleFull

```
[play-ws] [info] CachingSpec
[play-ws] [info] 
[play-ws] [info] GET should
[play-ws] [error]   ! work once
[play-ws] [error]    java.lang.NullPointerException: null (CachingAsyncHttpClient.scala:74)
```

playframework/play-ws@ed7fcabf80aafd9b7a60a0c8db489e7f1620eaa0 seems like a likely culprit? /cc @wsargent

**Comment by SethTisue** [comment]
froze play-ws in dcdc03f231a4da9a052d64738c94776c1aae012c; test run https://scala-ci.typesafe.com/view/All/job/scala-2.12.x-integrate-community-build/1613/consoleFull

**Comment by SethTisue** [comment]
yup, green again

**Comment by wsargent** [comment]
Anything I need to do here?

**Comment by SethTisue** [comment]
@wsargent it's y'all's test, I have no idea what the failure means... up to you whether it's something you ought to investigate. (from the Scala team's perspective, all else being equal, having some projects in the community build be frozen like this isn't a problem)

sometimes a thing like this is early notice of a real regression, and ignoring it would be a missed opportunity.

other times, it's just that dbuild is using newer source dependencies than the binary dependencies in your own CI, and the problem will go away once all the repos involved have upgraded and are back on the same page.

**Comment by wsargent** [comment]
I just released play-ws 1.0.0 so it's probably that

**Comment by SethTisue** [comment]
seen again at https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1755/console when I tried unfreezing again

**Comment by SethTisue** [comment]
resolved differently now in eccacc3d679b2e64d09e7036fb770608da2bf7ac — rather than freezing play-ws, we no longer compile or run its `integration-tests` project, which is the one with the failing test. fyi @gmethvin
